content/events/berlin-blockchain-week-crypto-info-brunch.md

```diff
@@ -0,0 +1,46 @@
+---
+eventName: "Berlin Blockchain Week - Crypto Infra Brunch" #the name of your track or event and its mandatory
+
+eventDescription: "This will be a buffet of ideas and innovations from the new kids + industry-legends in German web3." #short description of your track or event limiting to 100-150 characters
+
+#Topics will show up as labels on the event card.Only a maximum of 4 tags will be displayed on the event card. Some references for topics - Blockchain, Web3, Cryptocurrency, Tech Talks, Workshop, etc.
+
+eventTopic: 
+  - "DePIN"
+  - "Web3 Infra"
+  - "<Topic_3>"
+  - "<Topic_4>" 
+
+eventType: "Social" #Type of the event: Please choose one among the below options or just leave it blank
+
+location: "Berlin, Germany"  #Specify the location of the event.If you aren't sure about the location then mention "Location TBD"
+
+venueName: "Full Node" #the event venue name (will show up on the event card) or just leave it blank
+
+venueAddress: "Glogauer Str. 6, 10999 Berlin, Germany" #the event venue address (will show up on a map) or just leave it blank
+
+venueMapsLink: "[<Google_Map_link_for_the_venue>](https://maps.app.goo.gl/P1UjhUywPmMxpP3C7)" #the event venue Map link (will show up on a map) or just leave it blank
+
+website: "[<your_website_url>](https://lu.ma/infrabrunch)" #make sure to have all the relevant information: dates, venue, program, ticketing (if any), etc. or just leave it blank
+
+startDate: "05/22/2024" # the start date of the event , date format is MM/DD/YYYY eg: if it is February 16th 2023 => 02/16/2023
+
+endDate: "05/22/2024" #the end date of the event , date format is MM/DD/YYYY eg: if it is February 18th 2023 => 02/18/2023
+
+tag: "Industry Event" #Mention any one of the event types. Don't leave it blank.
+
+dateTBD: false #If you don't have concrete dates, add the approximate dates & set dateTBD: true.
+
+#Preferred means of contacts - this is a placeholder for email(for eg:  - email|mailto:<email_id>) and other social handles like Twitter, LinkedIn, Discord, etc. (for eg.   - 'twitter|https://twitter.com/IPFS/status/1629199396700098560?s=20')
+
+preferredContacts:
+  - '<contact_means>|<contact_url>'
+
+#Event host names and their respective logos(preferred size is 48px width, 48px height)-place the logo file on the path 'public/uploads' for eg.   - IPFS|ipfs-logo.png
+
+eventHosts:
+  - Impossible Cloud|<host_logo>
```

Comment:
@laeccles I can see 4 hosts - Impossible Cloud, Gnosis, Hats Finance and HV Capital. Can yu please mention which host does the above-shared logo is linked to? Also please provide logos for the other remaining 3 hosts.  Cc @pandeyanujk

Reply:
The host previously linked is Impossible Cloud